traffic_ops/app/db/squash_migrations.sh

```diff
@@ -25,6 +25,7 @@ last_squashed_migration="$(<<<"$migrations_to_squash" tail -n1)"
 last_squashed_migration_timestamp="$(<<<"$last_squashed_migration" sed -E 's|migrations/([0-9]+).*|\1|')"
 first_migration="$(ls migrations/*.sql | grep -A1 "/${last_squashed_migration_timestamp}_" | tail -n1)"
 first_migration_timestamp="$(<<<"$first_migration" sed -E 's|migrations/([0-9]+).*|\1|')"
+# If running on mac, you need BSD version of sed. So, change -i to -i.bak for the below three sed statements.
 sed -i '/^--/,$d' create_tables.sql # keeps the Apache License 2.0 header
```

Comment:
Just modify the `sed` command, the modified command works on both the GNU and BSD versions of `sed`.